docs/source/03_tutorial/04_create_pipelines.md

```diff
@@ -505,11 +556,13 @@ The `data_processing_pipeline` will preprocess the data, and `data_science_pipel
 ```
```

Comment:
Again not your change, but the above step isn't required if the user just followed the tutorial and ran `kedro pipeline create data_science`, I'd suggest removing it as it's a bit confusing.

Reply:
I'm not sure what this is referring to - perhaps GH being weird. Do you remember?

Reply:
It was the step about creating an `__init__.py` file "As with the data processing modular pipeline, you should ensure that the data science modular pipeline is a Python package. To do so, create the file src/kedro_tutorial/pipelines/data_science/__init__.py containing the following:"

Reply:
Done!